game/src/contrib/components/InventoryComponent.java

```diff
@@ -54,14 +54,22 @@ public InventoryComponent(int maxSize) {
      * @return True if the item was added, false if not.
      */
     public boolean add(final ItemData itemData) {
-        if (inventory.size() >= maxSize) return false;
+        int firstEmpty = -1;
+        for (int i = 0; i < this.inventory.length; i++) {
```

Comment:
Wenn ich mir das so anschaue, wäre eine List nicht deutlich angenehmer zum handeln?

Reply:
Liste ist doof, wegen leerer Slots zwischendurch. Und wir müssen früher oder später sowieso über die Liste iterieren, da die Möglichkeit Items zu stacken in der `ItemData`-Klasse mit `count` bereits vorbereitet ist und wahrscheinlich dann durch das Umarbeiten der Items mit implementiert wird.